packages/main/src/DatePicker.js

```diff
@@ -817,7 +843,7 @@ class DatePicker extends UI5Element {
 		minDate.setYear(1);
 		minDate.setMonth(0);
 		minDate.setDate(1);
-		return minDate.valueOf();
+		return minDate.toUTCJSDate().valueOf();
```

Comment:
Why is that? Doesn't they return the same result?

Reply:
Yes. The result is the same in both cases. I just liked to indicate that it is an UTC date.